.gitignore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
+.vscode
```

Comment:
These patterns are not specific to this project and should not be in the repo `.gitignore`. You should put them into your `~/.gitignore` file instead.